index.html

```diff
@@ -18639,14 +18578,16 @@ <h3>
             <dl class="attributes" data-dfn-for="AudioWorkletNode"
             data-link-for="AudioWorkletNode">
               <dt>
-                <code><dfn>onprocessorstatechange</dfn></code> of type
-                <span class=
+                <code><dfn>onprocessorerror</dfn></code> of type <span class=
                 "idlAttrType"><a><code>EventHandler</code></a></span>
               </dt>
               <dd>
-                Any state change on the processor will queue a task on the
-                control thread to fire <a>onprocessorstatechange</a> event to
-                the node.
+                When an exception is thrown from the processor's
+                <code>constructor</code>, <code>process</code> method, or any
+                user-defined class method, the processor will queue a task on
+                the control thread to fire <a>onprocessorerror</a> event to the
+                node. Note that once an exception is thrown, the processor will
+                output silence throughout its lifetime.
```

Comment:
Is any information about what caused the error included in the event?  That would be great if possible so that the developer has some clue on what happened.

Reply:
This should work similarly to `Worker.onerror`.
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/onerror

The EventHandler will fire the callback with `error` object as an argument.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error

The error message can vary between implementations, but the error type must be consistent and it should be specified in the relevant section. (i.e. AWP.constructor or AWP.process)